README.md

```diff
@@ -581,6 +583,30 @@ This document is written to capture the conventions of writing styles for our pr
 ### Disclaimer
 It's important to note that client and project requirements always undo choices made in these guidelines
 
+
+### Styleguide implementation
+
+#### Colors
+Store colors centrally as variables. Preferably use human-friendly names that make communication between departments easier. In many cases clients will provide brand guidelines with names for colors. Use these when possible in all documentation and communication.
+
+When using custom properties in css it's still recommended using variables.
+```scss
+// centrally stored
+$gingerOrange: #FF6200;
+$chamois: #DAA06D;
+
+:root {
+  --card-background: #{$chamois};
+}
+```
+If needed it's fine to prefix colors with `color`, but the majority of developers didn't see the need to make it a hard required. The recommendation is to stay consistent in its use.
+```scss
+$colorGingerOrange: #FF6200;
+$colorChamois: #DAA06D;
+```
+
+Note: Colors don't have to be RGB(A) or Hex. If there is an option/need for HSL to make use of different shades of saturation and/or lightness, HSL can be a very useful option. Perhaps the lightness can be adjusted for a dark/light mode without having to configure more than the lightness. There are also more color spaces available for certain devices. It might be nice to explore these options to enrich experiences. These are things you can discuss with e.g. designers and project managers. Keep in mind providing decent fallback colors if you are using the color() functional notation e.g.
```

Comment:
I think the color functional notation should only be used when the creatives use colours outside the sRGB color space. Using the color functional notation or notations like lab() and lch() for gradients should be allowed because different coordinate systems will create different gradients.

Reply:
This section should indeed be rewritten, something like:

- By default, use hex / rhg(a).
- For use case x or y, you are allowed to use hsl/lab/lch/etc because of <reasons>, but be aware of the <downsides>.

Reply:
The goal of this note is to inform people that we should not kill the use of other options (like hsl) or not provide enriched experiences by using `color()` by enforcing hex/rgb(a) as THE standard.